### PR TITLE
Change the position of SystemSlot and AuthoritySlot in soltchain

### PR DESCRIPTION
**Pull request by complone**
### Does this pull request fix one issue?
Change the position of SystemSlot and AuthoritySlot in soltchain

### CODE REVIEW

File: sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/slot/GatewaySlotChainBuilder.java [hunk 1]
```diff
@@ -44,8 +44,8 @@ public ProcessorSlotChain build() {
         chain.addLast(new LogSlot());
         chain.addLast(new StatisticSlot());
         // Rule checking slot
-        chain.addLast(new AuthoritySlot());
         chain.addLast(new SystemSlot());
+        chain.addLast(new AuthoritySlot());
```
Comment (sczyh30) [inline sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/slot/GatewaySlotChainBuilder.java]: In fact, the order of the slots in the `GatewaySlotChainBuilder` is more appropriate. Maybe we could update the `DefaultSlotChainBuilder`.

**Comment by sczyh30** [comment]
Related to #1052

File: sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/slot/GatewaySlotChainBuilder.java (hunk 1 above)
Comment (complone) [reply sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/slot/GatewaySlotChainBuilder.java]: Okay, it's been corrected.

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**